central/activecomponent/updater/updater_impl.go

```diff
@@ -144,6 +144,7 @@ func (u *updaterImpl) updateForDeployment(ctx context.Context, deploymentID stri
 
 		result, ok := u.executableCache.Get(update.ImageID)
 		if !ok {
+			//#nosec G104
```

Comment:
I think we should have a follow up PR to replace utils.Should with function that does not return error. Adding `#nosec` directives makes no sense

Reply:
There are tens of places where the return value is used. So there are two options: either implement two functions, or change the occurrences in this PR to `_ = utils.Should(...`.